src/components/tab/FileTabs.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="col" data-cy="file-tabs">
+    <q-tabs
+      v-model="internalActiveFile"
+      dense
+      class="text-grey bg-grey-2 tab-container"
+      active-color="primary"
+      indicator-color="transparent"
+      align="left"
+    >
+      <q-tab
+        v-for="file in files"
+        :key="file.id"
+        :name="file.id"
+        no-caps
+        :ripple="false"
+        class="tab-item"
+        :class="internalActiveFile === file.id ? 'tab-item--active' : 'tab-item--inactive'"
+        data-cy="file-tab-container"
+      >
+        <div class="row items-center">
+          <span
+            :class="internalActiveFile === file.id ? 'text-bold' : ''"
+            data-cy="file-tab-label"
+          >
+            {{file.label}}
+          </span>
+          <q-btn
+            flat
+            round
+            size="xs"
+            :color="internalActiveFile === file.id ? 'primary' : 'gray'"
+            icon="fa-solid fa-xmark"
+            class="q-ml-sm"
+            @click.stop="emit('update:close-file', file.id)"
+            data-cy="close-file-tab"
+          />
+        </div>
+      </q-tab>
+    </q-tabs>
+
+    <q-tab-panels v-model="internalActiveFile">
+      <q-tab-panel
+        v-for="file in files"
+        :key="file.id"
+        :name="file.id"
```

Comment:
add a data-cy to get file content, only for current file

Reply:
changed, to check